pkg/workflows/wasm/host/test/nodag/cmd/main.go

```diff
@@ -0,0 +1,36 @@
+//go:build wasip1
+
+package main
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/protoc/pkg/testdata/fixtures/capabilities/basicaction"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/protoc/pkg/testdata/fixtures/capabilities/basictrigger"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/v2"
+)
+
+func main() {
+	runner := wasm.NewDonRunner()
+	basic := &basictrigger.Basic{}
+	sdk.SubscribeToDonTrigger(runner, basic.Trigger(&basictrigger.Config{
```

Comment:
what kind of trigger is this analogous to IRL?

Reply:
It could be any DON trigger. ChainEvent, Cron, HTTP (if we go with a trusted Gateway, or signed payload). All the triggers we are talking about for SmartCon would be real examples.